sharktank/requirements.txt

```diff
@@ -5,7 +5,7 @@ gguf==0.10.0
 numpy<2.0
 
 # Needed for newer gguf versions (TODO: remove when gguf package includes this)
-# sentencepiece>=0.1.98,<=0.2.0
+sentencepiece>=0.1.98,<=0.2.0
```

Comment:
:/ looks like `gguf==0.10.0` still only includes these deps:

```
Requires-Dist: numpy (>=1.17)
Requires-Dist: pyyaml (>=5.1)
Requires-Dist: tqdm (>=4.27)
```
despite the inclusion in requirements: https://github.com/ggerganov/llama.cpp/blob/1a31d0dc00ba946d448e16ecc915ce5e8355994e/gguf-py/pyproject.toml#L21-L26

I didn't see an upstream issue at a glance, but we should file one and follow up upstream instead of carrying around these downstream patches forever.

Reply:
Started typing out an issue, but realized that the pyproject.toml linked above is included in [v0.11.0](https://github.com/ggerganov/llama.cpp/blob/gguf-v0.10.0/gguf-py/pyproject.toml), but not in [v0.10.0](https://github.com/ggerganov/llama.cpp/blob/gguf-v0.10.0/gguf-py/pyproject.toml).

Looks like [v0.11.0](https://pypi.org/project/gguf/#history) was released an hour ago. Thinking we could try an upgrade?

Reply:
Yes let's try upgrading. The metadata is there in the .whl file now.

```
Requires-Dist: numpy (>=1.17)
Requires-Dist: pyyaml (>=5.1)
Requires-Dist: sentencepiece (>=0.1.98,<=0.2.0)
Requires-Dist: tqdm (>=4.27)
```

Reply:
CI and doc walkthrough passed using `gguf=0.11.0`